tosca-samples/org/ystia/yorc/samples/job/noscheduler/components/scripts/submit.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+echo "Submitting job..."
+export TOSCA_JOB_ID="Job-${RANDOM}"
```

Comment:
Why exporting this?

Reply:
Unused line coming from a first try, will remove it, it is the export done below that is used